src/ts/components/Controls.tsx

```diff
@@ -33,45 +32,58 @@ interface IState {
  */
 class Controls extends Component<IProps, IState> {
   public state = {
-    isOpen: false,
     allCategories: [],
     currentCategories: [],
     isLoadingCategories: false,
-    pluginState: undefined,
+    pluginState: undefined
   } as IState;
   public componentWillMount() {
     this.props.store.on(STORE_EVENT_NEW_STATE, this.handleNotify);
+    this.setState({ pluginState: this.props.store.getState() });
     this.initCategories();
   }
 
   public render() {
+    const checkInProgress: boolean | undefined =
+      this.state.pluginState &&
```

Comment:
I think accessing this plugin state could be better achieved through the usage of selectors.